lib/RTCSession.js

```diff
@@ -942,6 +942,7 @@ module.exports = class RTCSession extends EventEmitter
     let position = 0;
     let duration = options.duration || null;
     let interToneGap = options.interToneGap || null;
+    const transportType = options.transportType || JsSIP_C.DTMF_TRANSPORT.INFO;
```

Comment:
After line 945, please validate `transportType`. If not a valid one, throw a `TypeError`.

Reply:
done